test/test.js

```diff
@@ -1,9 +1,13 @@
-const assert = require('assert');
+
```

Comment:
We use empty lines as separators, but there is nothing to separate at the start of a file. I think we can remove it.